build.gradle

```diff
@@ -338,7 +338,7 @@ subprojects {
 
             check.dependsOn("testModules")
 
-            if (!(project.name in ['micrometer-jakarta9'])) { // add projects here that do not exist in the previous minor so should be excluded from japicmp
+            if (!(project.name in ['micrometer-jakarta9', 'micrometer-registry-otlp'])) { // add projects here that do not exist in the previous minor so should be excluded from japicmp
```

Comment:
In the `japicmp` task configuration, you can make more fine-grained excludes, e.g. a specific method.